parser_library/test/workspace/b4g_integration_test.cpp

```diff
@@ -25,6 +25,23 @@ using namespace hlasm_plugin::parser_library;
 using namespace hlasm_plugin::parser_library::workspaces;
 using hlasm_plugin::utils::resource::resource_location;
 
+namespace {
+const resource_location pgm_a("SYS/SUB/ASMPGM/A");
+const resource_location pgm_b("SYS/SUB/ASMPGM/B");
+const resource_location mac1("SYS/SUB/ASMMAC/MAC1");
+const resource_location mac1_p1("SYS/SUB/ASMMACP1/MAC1");
+const resource_location mac1_p2("SYS/SUB/ASMMACP2/MAC1");
```

Comment:
maybe move `b4g_conf_name` and `empty_b4g_conf` here? as they are only used in this test file.